src/meta/src/barrier/progress.rs

```diff
@@ -299,9 +322,11 @@ impl CreateMviewProgressTracker {
     pub fn recover(
         mviews: impl IntoIterator<Item = (TableId, (String, &StreamJobFragments))>,
         version_stats: &HummockVersionStats,
+        mut backfill_order_map: HashMap<TableId, BackfillOrderState>,
```

Comment:
Better to include the `BackfillOrderState` to the `mviews` iter.

Reply:
https://github.com/risingwavelabs/risingwave/pull/20967/commits/2f8679b5652c3783d55d3ac19bf95d3e0fa890a1 removed recovery logic, so we no longer have this paramter